src/translations/translations.en.js

```diff
@@ -632,6 +632,7 @@ export const translations = {
   },
   //#endregion
   //#region cc-pricing-header
+  'cc-pricing-header.error': `Something went wrong while loading pricing filters.`,
```

Comment:
Make it more like other error messages:

```suggestion
  'cc-pricing-header.error': `An error occured while loading pricing filters.`,
```

Reply:
Nice catch although we actually use both in our components.
It looks like we use the "Something went wrong" more often but it does not mean it's the best?

Both are fine for me so we should choose one and change it everywhere in a separate PR, I'll open an issue :+1: 

Thanks @pdesoyres-cc!